lib/shopify_app/controller.rb

```diff
@@ -3,16 +3,6 @@
 module ShopifyApp
   module Controller
     extend ActiveSupport::Concern
-
     include ShopifyApp::LoginProtection
-
-    def fullpage_redirect_to(url)
-      if ShopifyApp.configuration.embedded_app?
-        render inline: %Q(<script type="text/javascript">window.top.location.href = #{url.to_json};</script>)
-      else
-        redirect_to url
-      end
-    end
-
```

Comment:
I think we should remove this - this repo is moving more towards a proper rails engine from the old generator and it provides controllers now. 

What you'll need to do is replace any includes of ShopifyApp::Controller with ShopifyApp::LoginProtection. Also make a note of that in the CHANGELOG since downstream users will need to make this change when they update. Their is already module renames in this major version